aemedge/blocks/featured-articles/featured-articles.css

```diff
@@ -1,10 +1,35 @@
 .featured-articles {
   & ul {
     display: grid;
-    grid-auto-flow: row;
-    grid-auto-rows: 1fr;
-    grid-template-columns: 3fr 2fr;
+    grid-template-columns: repeat(auto-fill, minmax(278px, 1fr));
```

Comment:
@saurabh-khare : Where does this value come from?

Reply:
Its historical and I copied over from @benpeter's code when I refactored the content-list. IMHO it can go but can be deferred till the presentation so we don't break anything with cards.

Reply:
Ok